feat(get): Add 'get rds' command

amalmadhu06 opened the pull request:
Added get rds command which retrieves useful information related to RDS instances such as Instance ID, Engine, Status and Endpoint. 

## Fixes Issue
Closes #17 

## Changes proposed

<!-- List all the proposed changes in your PR -->
Added get rds command

## Screenshots

![image](https://user-images.githubusercontent.com/46180807/227013886-8a277568-23bb-4bd5-abcd-a3f09a74209b.png)


## Note to reviewers
I have added Instance ID, Engine, Status, and Endpoint in the output assuming that they are useful information.

surajnarwade commented on pkg/getRdsInfo.go:
can this go inside `/pkg/rds/rds.go`

amalmadhu06 replied:
Sure. On it

amalmadhu06 commented:
- Removed pkg.getRdsInfo.go,  moved code to pkg/rds.rds.go.
- Added  VERSION in output

![image](https://user-images.githubusercontent.com/46180807/227133474-de0c2e43-4afa-4945-8a47-cfd82d909e1b.png)

surajnarwade commented on cmd/rds.go:
```diff
@@ -0,0 +1,51 @@
+	w := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
+	defer w.Flush()
+
+	fmt.Fprintln(w, "DB_INSTANCE_ID\tSTATUS\tENDPOINT\tENGINE\tVERSION")
```
this will be slightly different, check other commands

amalmadhu06 replied:
changed
 `fmt.Fprintln(w, "DB_INSTANCE_ID\tSTATUS\tENDPOINT\tENGINE\tVERSION")` 
to 
`fmt.Fprintln(w, "DB_INSTANCE_ID", "\t", "STATUS", "\t", "ENDPOINT", "\t", "ENGINE", "\t", "VERSION")`

surajnarwade commented on cmd/rds.go:
```diff
@@ -0,0 +1,51 @@
+	fmt.Fprintln(w, "DB_INSTANCE_ID", "\t", "STATUS", "\t", "ENDPOINT", "\t", "ENGINE", "\t", "VERSION")
+
+	for _, i := range rdsInstances {
+		fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\n", i.DBInstanceID, i.Status, i.Endpoint, i.Engine, i.Version)
```
can you do the same for this as well, it makes code easier to read.

amalmadhu06 replied:
Removed  `fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\n", i.DBInstanceID, i.Status, i.Endpoint, i.Engine, i.Version)` and included 
`fmt.Fprintln(
			w,
			i.DBInstanceID, "\t",
			i.Status, "\t",
			i.Endpoint, "\t",
			i.Engine, "\t",
			i.Version, "\t",
		)`

surajnarwade reviewed (approved):
Thank you for your contribution @amalmadhu06 this LGTM 🎉
